client/tests/test_client.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from client import models
+from client.models import Client
+from django.core.exceptions import ValidationError
+
+
+USER_NAME_TEST = 'liorsil'
+FIRST_NAME_TEST = 'lior'
+LAST_NAME_TEST = 'silber'
+PASSWORD_TEST = '1234qwer'
+USER_TYPE_TEST = 'client'
+AREA_TEST = 'yuvalim'
+
+
+@pytest.fixture
+def client0():
+    return models.Client(user_name=USER_NAME_TEST,
+                         first_name=FIRST_NAME_TEST,
+                         last_name=LAST_NAME_TEST,
+                         password=PASSWORD_TEST,
+                         user_type=USER_TYPE_TEST,
+                         area=AREA_TEST)
+
+
+@pytest.fixture
+def saved_client0(client0):
+    client0.save()
+    return client0
+
+
+class TestClientModel:
+    @pytest.mark.django_db()
+    def test_save_client(self, client0):
+        assert client0 not in models.Client.objects.all()
+        Client.save_client(client0)
+        assert client0 in models.Client.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_client(self, saved_client0):
+        assert saved_client0 in Client.objects.all()
+        Client.delete_client(saved_client0)
+        assert saved_client0 not in Client.objects.all()
+        # try to delete client that not exist
+        with pytest.raises(ValueError):
+            Client.delete_client(saved_client0)
+
+    @pytest.mark.django_db()
+    def test_get_all_clients(self):
+        assert len(Client.get_all_clients()) != 0
+        for cli in Client.get_all_clients():
+            assert cli in Client.objects.all()
+
+    @pytest.mark.django_db()
+    def test_fail_to_add_client(self, client0):
+        short_user_name = 'lior'
+        empty_field = ''
+        short_password = '123'
```

Comment:
How do you know which of them failed you? Suppose that tomorrow someone accidentally breaks the validator for the password field, this test will still pass. You need to have each failure on its own test (you can also parameterize this test if you prefer - https://docs.pytest.org/en/6.2.x/parametrize.html).